### PR TITLE
fixed potential memory leak

### PR DESCRIPTION
**Pull request by cyberNinja6143**
# Fixed a potential memory leak while investigating an issue a user was complaining about.

# Changes proposed in this pull request:
- Removed a variable that was not being used

[ *] Yes I tested this before submitting it.
[ ] I also did a multiplayer test.

Review please

### CODE REVIEW

**Comment by someaddons** [comment]
in java that is not a memory leak :D but we can remove the line if it isnt used

**Comment by cyberNinja6143** [comment]
> in java that is not a memory leak :D but we can remove the line if it isnt used

Yeah but it still looks bad in the source code

**Comment by someaddons** [comment]
I'm less concerned about visuals in code

**Comment by cyberNinja6143** [comment]
> I'm less concerned about visuals in code

It is a bit of a silly change but I'm kind of a new developer

**Comment by Thodor12** [comment]
When you make contributions you have to follow our style guide, all code has to be formatted with our formatting rules.
The space you removed is exactly against our formatting rules. So that has to be removed.

**Comment by cyberNinja6143** [comment]
> When you make contributions you have to follow our style guide, all code has to be formatted with our formatting rules. The space you removed is exactly against our formatting rules. So that has to be removed.

My bad, ill fix it

**Review by Thodor12 — DISMISSED** [review]

File: src/main/java/com/minecolonies/core/entity/ai/minimal/EntityAIInteractToggleAble.java [hunk 1]
```diff
@@ -387,7 +387,7 @@ public void tick()
         {
             return;
         }
-        updateTimer = ColonyConstants.rand.nextInt(40 + offSet);
+        updateTimer = ColonyConstants.rand.nextInt(30 - offSet);
```
Comment (Raycoms) [inline src/main/java/com/minecolonies/core/entity/ai/minimal/EntityAIInteractToggleAble.java:390]: if you want to update this please make this a separate PR
